rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/360_date_histogram.yml

```diff
@@ -356,3 +385,58 @@ setup:
 
   - match: { hits.total.value: 1 }
   - length: { aggregations.datehisto.buckets: 3 }
+
+---
+"Daylight with offset date_histogram test":
+  - skip:
+      version: "- 8.2.99"
```

Comment:
I know this is incorrect...I will try to see if I can find when this was fixed (assuming the test results are correct)

Reply:
As noted, I think 7.17 is good enough.